example/lib/main.dart

```diff
@@ -30,10 +31,10 @@ class MyHomePage extends StatefulWidget {
   final String title;
 
   @override
-  _MyHomePageState createState() => _MyHomePageState();
+  MyHomePageState createState() => MyHomePageState();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
+class MyHomePageState extends State<MyHomePage> {
```

Comment:
why public?

Reply:
I'm not really sure tbh. I'll see if I can tidy it up later today.
I have a feeling my IDE may have been acting up, as I can't possibly think why I did this.

Reply:
Resolved in da3eb05.